server/fig.yml

```diff
@@ -1,41 +0,0 @@
-db:
```

Comment:
Good point, I have to update the README.md.
We don't use fig anymore but docker-compose. And the template file is in `docker/`.
Anyway we need a better README, I did this one 5s before the initial commit on the new repo.